data/hai outfits.txt

```diff
@@ -600,3 +600,12 @@ outfit `"Bufaer" Atomic Steering`
 	"turning energy" 5.2
 	"turning heat" 13.8
 	description "Nine out of ten human captains interviewed in Hai space say that they would choose Hai atomic engines over those of Deep Sky any day... that is, if they could afford such expensive engines."
+
+outfit "Pulse Rifle"
+	category "Hand to Hand"
+	cost 11000
+	thumbnail "outfit/hai rifle"
+	"capture attack" .8
+	"capture defense" 1.3
+	"unplunderable" 1
+	description "The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids, and while its since been adapted to not be lethal outright if enough bolts hit in a short period of time the nerves will fry, shutting down the body and killing the target."
```

Comment:
first of all, this single sentence is way too long and confusing
and "and while its since been adapted to not be lethal outright if enough bolts hit in a short period of time the nerves will fry" sounds incorrect and probably is

maybe try
"The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to not be lethal outright, if several bolts hit a target rapidly, it will shut down the nervous system, killing the target."

Reply:
yeah thanks im not the best at sentence setup